### DISCUSSION TITLE
Can't find AddHttpService feature that looks like it should have been included in preview 2

### DISCUSSION THREAD

**Discussion by RobLofthouseAbsa** [microsoft/aspire#1588] [General]
HI all, I might be missing something extremely obvious here, but looking at PR https://github.com/dotnet/aspire/pull/1030, it is marked as merged and should by all accounts (at least as far as I can see) should be in the prerelease 2 build, but I'm referencing nuget package version 8.0.0-preview.2.23619.3, and I can't actually find it. Looking at the prerelease2 or the main branches, the code changes don't seem to be there either. 

This will be a game changer in the potential to use Aspire for us, so I might that all involved in getting this work done so super quickly, but I can't seem to use it yet.

**Comment by davidfowl** [comment]
This ended up being a call to WithReference that takes a Uri instead of a resource type.

**Reply by RobLofthouseAbsa** [reply]
Ok thanks, I'll ferret around to see what the new implementation is like.

**Reply by RobLofthouseAbsa** [reply]
Right ... got it, much better, thanks. Will close this discussion now.
